src/data-pipeline/clean.py

```diff
@@ -205,7 +205,7 @@ def clean_one_study(study):
 
     # Inverventions module
     interv_module = protocols.get("armsInterventionsModule", {})
-    cleaned_data["Interventions"] = [
+    cleaned_data["interventions"] = [
```

Comment:
This is for consistency with the top comment and for consisteny with other entries - we are always using lower snake case for the keys. This would require to rerun the whole pipeline, but that's fine.